pkg/netflow/config/config.go

```diff
@@ -21,6 +21,7 @@ type NetflowConfig struct {
 	StopTimeout                   int              `mapstructure:"stop_timeout"`
 	AggregatorBufferSize          int              `mapstructure:"aggregator_buffer_size"`
 	AggregatorFlushInterval       int              `mapstructure:"aggregator_flush_interval"`
+	AggregatorFlushToSendInterval int              `mapstructure:"aggregator_flush_to_send_interval"`
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Do we have a use case where it might be useful to users to configure `AggregatorFlushToSendInterval` ? If not, I think it might be better to keep it internal for now since it gives us more flexibility to changes things (once we expose a config, it's hard to remove/change).

Reply:
The only use case I'm aware of is ourselves - otherwise our integration tests have to wait for this to tick faster. I can expose it to the tests without putting it in the config, though, at least in the componentized version.

Reply:
I've removed the field from this PR, and added a hack to set it to a small number in the component PR.